### PR TITLE
Generate pythons' FDK boilerplate

### LINKED ISSUE #111: Generate FDK boilerplate for python
The CLI should generate boilerplate with a basic example on fn init (as per java) for go and python. 

* Should honor "FDK_VERSION" env option (or CLI option) to select FDK version explicitly 
* Should get latest version remotely and fix this in the func.yaml/code if not set
* Should include a basic "hello world" example
* Should include a fn test

### PR DESCRIPTION
**Pull request by reclaro**
CLI now generates a basic example on fn init.
The example includes a fn test.
Closes #111

### CODE REVIEW

File: langs/python.go [hunk 1]
```diff
@@ -64,6 +67,85 @@ func (h *PythonLangHelper) IsMultiStage() bool {
...
+	if err := ioutil.WriteFile(testFile, []byte(pythonTestBoilerPlate), os.FileMode(0644)); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+const (
+	helloPythonSrcBoilerplate = `
+import sys
```
Comment (denismakogon) [inline langs/python.go]: This boilerplate is wrong for anything except default format. This type of the boilerplate works only for one particular case - non-fdk default format. The boilerplate should be generated with respect to https://github.com/fnproject/fdk-python. But that is not all, fdk-python would never support python2, so if the python runtime version is 2.x CLI should fail.

**Comment by treeder** [comment]
It's OK for FDK's to just support a single hot format (preferably json). They don't need to support the default format at all (since FDK's don't provide much value on top of default format).

**Comment by treeder** [comment]
I'm OK with dropping 2.0 support?  Any really good reason to keep it?

**Comment by denismakogon** [comment]
Nope, there’s no point to keep that.

**Comment by carimura** [comment]
also want to link to this one -- separate but related while in there: https://github.com/fnproject/cli/issues/194

**Comment by denismakogon** [comment]
Sorry for overtaking this one. Closing per https://github.com/fnproject/cli/pull/195

**Pull request closed without merging**